docs/docs/dev/deployments/multi-zone.md

```diff
@@ -532,5 +455,57 @@ When it happens:
 * Communication across each zone will fail.
 
 ::: tip
-With the right resiliency setup ([Retries](../../policies/retry), [Probes](../../policies/health-check), [Locality Aware LoadBalancing](../../policies/locality-aware), [Circuit Breakers](../../policies/circuit-breaker)) the failing zone can be quickly severed and traffic re-routed to another zone.
+With the right resiliency setup ([Retries](../policies/retry.md), [Probes](../policies/health-check.md), [Locality Aware LoadBalancing](../policies/locality-aware.md), [Circuit Breakers](../policies/circuit-breaker.md)) the failing zone can be quickly severed and traffic re-routed to another zone.
+:::
+
+## Delete a zone
+
+To delete a `Zone` we must first shut down the corresponding Kuma zone control plane instances. As long as the Remote CP is running this will not be possible, and Kuma returns a validation error like:
```

Comment:
```suggestion
To delete a `Zone` we must first shut down the corresponding Kuma zone control plane instances. As long as the Zone CP is running this will not be possible, and Kuma returns a validation error like:
```